overlay/packages/apps/CarrierConfig/res/xml/vendor.xml

```diff
@@ -60,1542 +60,32 @@ IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.-->
-        <int name="carrier_default_wfc_ims_mode_int" value="2" />
-    </carrier_config>
-    <carrier_config mcc="655" mnc="07">
-        <boolean name="carrier_wfc_ims_available_bool" value="true" />
-        <int name="carrier_default_wfc_ims_mode_int" value="2" />
-    </carrier_config>
-
-    <carrier_config mcc="655" mnc="07" spn="Cell C">
-        <boolean name="carrier_wfc_ims_available_bool" value="true" />
-        <int name="carrier_default_wfc_ims_mode_int" value="2" />
-    </carrier_config>
-    <carrier_config mcc="655" mnc="10">
-        <boolean name="carrier_wfc_ims_available_bool" value="true" />
-        <int name="carrier_default_wfc_ims_mode_int" value="2" />
-    </carrier_config>
-    <!-- Carrier-specific Movistar PE -->
-    <carrier_config mcc="716" mnc="06">
-        <boolean name="carrier_volte_available_bool" value="true" />
-        <boolean name="carrier_vt_available_bool" value="true" />
-    </carrier_config>
-
-    <!-- Carrier-specific Movistar AR -->
-    <carrier_config mcc="722" mnc="070">
-        <boolean name="carrier_volte_available_bool" value="true" />
-        <boolean name="carrier_wfc_ims_available_bool" value="true" />
-        <boolean name="carrier_vt_available_bool" value="true" />
-        <int name="carrier_default_wfc_ims_mode_int" value="1" />
-    </carrier_config>
-    <!-- Carrier-specific Claro AR -->
-    <carrier_config mcc="722" mnc="310">
-        <boolean name="carrier_volte_available_bool" value="true" />
-        <boolean name="carrier_wfc_ims_available_bool" value="true" />
-        <int name="carrier_default_wfc_ims_mode_int" value="2" />
-    </carrier_config>
-    <carrier_config mcc="722" mnc="320">
-        <boolean name="carrier_volte_available_bool" value="true" />
-        <boolean name="carrier_wfc_ims_available_bool" value="true" />
-        <int name="carrier_default_wfc_ims_mode_int" value="2" />
-    </carrier_config>
-    <carrier_config mcc="722" mnc="330">
-        <boolean name="carrier_volte_available_bool" value="true" />
-        <boolean name="carrier_wfc_ims_available_bool" value="true" />
-        <int name="carrier_default_wfc_ims_mode_int" value="2" />
-    </carrier_config>
-    <!-- Carrier-specific Movistar CO -->
-    <carrier_config mcc="732" mnc="102">
-        <boolean name="carrier_volte_available_bool" value="true" />
-    </carrier_config>
-    <carrier_config mcc="732" mnc="123">
-        <boolean name="carrier_volte_available_bool" value="true" />
+    
+    <carrier_config>
+	<!-- Make VoLTE, VoWiFi and Video Telephony "available" or "enabled by platform". -->	
+	<boolean name="carrier_volte_available_bool" value="true" />
+	<boolean name="carrier_wfc_ims_available_bool" value="true" />
+	<boolean name="carrier_vt_available_bool" value="true" />
+	<boolean name="use_rcs_presence_bool" value="true" />
+	
+	<!-- Enable provisioning, so we do not reach 'the end of the world' by unavailable features. -->
```

Comment:
What is meant be "reaching the end of the world" here and below?

Reply:
Downgrade techs if unavailable, or don't provision unavailable techs. Otherwise, user gets busy tone (end of the world - a joke if you will).